src/libs/navigation.c

```diff
@@ -454,6 +454,11 @@ static void _zoom_preset_change(int val)
     scale = 2.0f;
     zoom = DT_ZOOM_FREE;
   }
+  else if(val == 4)
```

Comment:
This long chain of `if-else-if` statements may be replaced by a `case` statement.